lib/handlers/hard-quote/handler.ts

```diff
@@ -47,21 +47,19 @@ export class QuoteHandler extends APIGLambdaHandler<
       throw new UnknownOrderCosignerError();
     }
 
-    // TODO: finalize on v2 metrics logging
+    // Instead of decoding the order, we rely on frontend passing in the requestId
+    //   from indicative quote
     log.info({
-      eventType: 'HardQuoteRequest',
+      eventType: 'HardRequest',
       body: {
         requestId: request.requestId,
-        tokenInChainId: request.tokenInChainId,
-        tokenOutChainId: request.tokenInChainId,
-        encoded: requestBody.encodedInnerOrder,
-        sig: requestBody.innerSig,
+        quoteId: request.quoteId,
```

Comment:
so `quoteId` here matches the indicative `requestId`?

Reply:
not sure if quoteId should exist in hard request tbh... I just saw it's part of the request currently